tests/naive.py

```diff
@@ -156,10 +156,13 @@ def stamp(T_A, m, T_B=None, exclusion_zone=None):
     return result
 
 
-def stump(T_A, m, T_B=None, exclusion_zone=None):
+def stump(T_A, m, T_B=None, exclusion_zone=None, row_wise=False):
     """
-    Traverse distance matrix along the diagonals and update the matrix profile and
+    Traverse distance matrix and update the matrix profile and
     matrix profile indices
+
+    If the parameter `row_wise` is set to `False`, then it is a diagonal traversal.
```

Comment:
The first letter should be lowercase in this `If` since it continues the sentence.

Reply:
I actually forgot to end the previous sentence with period (`.`)...So:

```
Traverse distance matrix and update the matrix profile and
matrix profile indices. # the sentence is  ended here

# new sentence
If ...
```

---

I think what you are suggesting is this:
```
Traverse distance matrix diagonally and update the matrix profile and 
matrix profile indices if the parameter `row_wise` is set to `False`. 
If the parameter `row_wise` is set `True`, then it is a row-wise traversal.
```

Should I go with the latter one?

Reply:
Sure